src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -53,9 +53,13 @@ function initStore() {
             // normalize cancer_study_list this handles legacy sessions/urls where queries with single study had different param name
             const cancer_study_list = query.cancer_study_list || query.cancer_study_id;
 
+            const cancerStudyIds = cancer_study_list.split(",");
```

Comment:
@zhx828 @kalletlak  we also have cohortIds parameter and this maybe the same as cancer study list?  When we submit a virtual study, does the cancer_study_list param contain all physical studies or just virtual study?  and what about cohortIdsList?

Reply:
@alisman it contains virtual study.

Reply:
anyone else think this is getting confusing and needs some documentation